server.js

```diff
@@ -0,0 +1,165 @@
+// dependencies
+var express = require("express");
+var bodyParser = require("body-parser");
+var logger = require("morgan");
+var mongoose = require("mongoose");
+// require note and article models
+var Note = require("./models/Note.js");
+var Article = require("./models/Article.js");
+// scraping tools
+var request = require("request");
+var cheerio = require("cheerio");
+// mongoose mpromise deprecated - use bluebird promises
+var Promise = require("bluebird");
+
+mongoose.Promise = Promise;
+
+// initialize express
+var app = express();
+
+// use morgan and body parser with our app
+app.use(logger("dev"));
+app.use(bodyParser.urlencoded({
+	extended: false
+}));
+
+// make public a static dir
+app.use(express.static("public"));
+var MONGODB_URI = process.env.MONGODB_URI || "mongodb://localhost/scrappy";
+var tempURI = "mongodb://heroku_0vmm96p8:asvcfubbgu4uf9q9n6udl95ivm@ds133876.mlab.com:33876/heroku_0vmm96p8";
+// database configuration with mongoose
+mongoose.connect(MONGODB_URI);
+var db = mongoose.connection;
+
+// show any mongoose errors
+db.on("error", function(error) {
+	console.log("Mongoose Error: ", error);
+});
+
+// once logged into the db through mongoose, log a success message
+db.once("open", function() {
+	console.log("Mongoose connection successful.");
+});
+
+// ROUTES
+
+// simple index route
+app.get("/", function(req, res) {
+	res.send(index.html);
+});
+
+// a get request to scrape the luckypeach website
+app.get("/scrape", function(req, res) {
+	// first, we grab the body of the html with request
+	request("http://www.theolympian.com", function(error, response, html) {
+		// then, we load that into cheerio and save it to $ for shorthand
+		var $ = cheerio.load(html);
+		// now, we grab every h2 within an article tag and do the following:
+		// $(".entry-title").each(function(i, element) {
+		$("h4.story-list").each(function(i, element) {
```

Comment:
The reason why you aren't scraping any data is that there are no `<h4>` elements with the class of "story-list".  You can confirm this by going to www.theolympian.com and entering `$("h4.story-list")` in the console.